### ISSUE TITLE
Error with .add_section()

### ISSUE THREAD

**Issue by BrianDORourke** [JWock82/Pynite#224]
**Describe the bug**
There seems to be a bug in FEModel3D.add_section()

A code snippet of # Example of a simply supported beam with a uniform distributed load is as follows : 
Iy = 100 #in^4 
Iz = 150 #in^4, 
J = 250 #in^4, 
A = 20 #in^2
simple_beam.add_section('MySection', Iy, Iz, J, A)

This runs to give the error:

TypeError: FEModel3D.add_section() takes 3 positional arguments but 6 were given

Any help would be appreciated. Thanks in advance. Brian O'Rourke

**Comment by JWock82** [comment]
I ran that code without any errors. When you initialize `simple_beam` are you forgetting the parentheses?

    from PyNite import FEModel3D
    simple_beam = FEModel3D()

**Comment by BrianDORourke** [comment]
Thanks for reply. Checked that and ran the code giving the same error. It must be something in my library installation.

**Comment by overagain** [comment]
yeah, i have same strange error. when I go to FEModel3D class in the updated version in the add_section function there are only 3 arguments, but in the source there should be 6. i know that it's the latest version

**Comment by bjhowie** [comment]
I believe this is due to changes pushed here #218 .
The `add_section` function has been updated to directly accept A, J,  Iz, Ix whereas previously you had to pass in an already created Section instance.

I don't believe the version on pypi reflects this change, so if that's the version you're using, you will need to manually create the Section instance and pass it to the `add_section` function.

**Comment by JWock82** [comment]
I'm running the development version of `Pynite` and having no issues. It sounds like I need to create a new release.

**Comment by JWock82** [comment]
I tried uploading the fix for this, but PyPi isn't playing nice. It's giving me a 403 error and telling me username and password authentication is no longer allowed, which is funny because I switched to API tokens long ago. So I tried generating a new API token and still no luck. Then I tried adding the repository to PyPi's list of trusted publishers. Still getting error 403. This is an issue on PyPi's side. I've got to step back and figure out my next step. Right now, due to a bug on PyPi's side, I can't upload new versions to PyPi. Aggravating.
